app/views.py

```diff
@@ -0,0 +1,107 @@
+from app import app
+from flask import render_template, request, redirect, jsonify, make_response, url_for
+from werkzeug.utils import secure_filename
+from base64 import b64encode
+import os
+import tensorflow as tf
+from hub.examples.image_retraining.label_image import get_labels, wiki
+from hub.examples.image_retraining.reverse_image_search import reverseImageSearch
+from flask_wtf import FlaskForm
+from flask_wtf.file import FileField, FileAllowed, FileStorage
+from wtforms import StringField, validators
+
+from PIL import Image
+import requests
+from io import BytesIO
+
+# no secret key set yet
+SECRET_KEY = os.urandom(32)
+app.config["SECRET_KEY"] = SECRET_KEY
+
+
+class SelectImageForm(FlaskForm):
+    image_url = StringField(
+        "image_url",
+        validators=[validators.Optional(), validators.URL()],
+        render_kw={"placeholder": "Enter a URL"},

```

Comment:
@Prakhar314 Fine with `Enter a URL` too, but any other suggestions to make it more cool?